guide/14-deep-learning/point_cloud_segmentation_using_pointcnn.ipynb

```diff
@@ -18,7 +18,7 @@
    "cell_type": "markdown",
```

Comment:
<ol><li>When training a PointCNN model using <code>arcgis.learn </code>, the raw point cloud dataset in LAS files is <s>first</s> converted into blocks of points <s>containing </s> <strong>that contains</strong> a specific number of points and their labels</li><li>We can also specify all the additional parameters while initializing PointCNN <s>in the form of</s> <strong>by </strong>using keyword arguments. The keyword argument takes in a parameter named&nbsp;<code>encoder_params</code>&nbsp;<strong>,</strong><s>keyword argument, in </s> which <s>we can be</s> <strong>is </strong>used to specify the number of output channels for each layer in the encoder</li><li>There are mainly four things that we need to define in the <s>encoder,  </s><strong>encoder i.e.</strong><code>out_channels</code>, P, K and D.</li><li>The <code>out_channel</code> specifies <strong>the </strong>number of channels after each layer</li><li><code>P</code>, <code>K</code> and <code>D</code> <s>specifiy </s> <strong>specify</strong> the number of points, k-nearest-neighbor and <strong>the </strong>dilation rate <strong>respectively </strong>in each layer</li><li>Additionally, one can specify <code>m</code> <strong>which</strong> is the multiplicative <s>factor, which</s> factor and is multiplied by each element of <strong>the</strong> out_channel list</li><li>The <code>sample_point_num</code> keyword argument specifies the number of points that we would like to train the model. <s>on</s></li><li>maximum points in a block <s>during </s>while exporting </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>''corrected accordingly'', Thanks.</p>